### ISSUE TITLE
Hyperparameter setting of GMM

### ISSUE THREAD

**Issue by RumengYi** [LiJunnan1992/DivideMix#20]
Hello, thanks for your excellent work!
Can two input losses be entered in the fit() of GMM? 
For example, gmm.fit(input_loss1, input_loss2)?
I am curious and want to try to use two losses to model GMM.

**Comment by LiJunnan1992** [comment]
Hi, thanks for your interest in our method!
You can concatenate the two losses and pass into fit(), please see here for how to use gmm.fit():
https://scikit-learn.org/stable/modules/generated/sklearn.mixture.GaussianMixture.html#sklearn.mixture.GaussianMixture.fit

**Comment by RumengYi** [comment]
Thank you very much! Could you please provide the reproduced code of F-correction?

**Comment by LiJunnan1992** [comment]
Hi, you can check out the author's implementation here: https://github.com/giorgiop/loss-correction
